PL2/PL2-core/src/test/java/nl/tudelft/pl2016gr2/core/algorithms/FilterSnipsTest.java

```diff
@@ -36,16 +51,111 @@ public void testMakeSnip() {
 
       graph.addNode(node);
     }
+  }
 
-    // Node zeroNode = new Node(0, 1, null, 0);
-    // zeroNode.addOutlink(1);
-    // graph.addNode(zeroNode);
-
-    graph.print();
-    System.out.println("-------------------");
+  /**
+   * Test filter method. The resulting graph should have 3 nodes:
+   *    - 8
+   * 1 -
+   *    - 9
+   */
+  @Test
+  public void testFilter() {
     FilterSnips filter = new FilterSnips(graph);
     OriginalGraph filteredGraph = filter.filter();
-    // filteredGraph.print();
+    
+    assertEquals(filteredGraph.getSize(), 3);
```

Comment:
The first parameter of `assertEquals` should be the expected value and the second parameter should be the actual value:
`assertEquals(3, filteredGraph.getSize());`